add-on/TA-Demisto/bin/ta_demisto/modalert_create_xsoar_incident_utils.py

```diff
@@ -37,14 +37,38 @@ def get_incident_labels(helper, event, labels_str, ignore_labels, search_query=N
 
 
 def get_incident_custom_fields(custom_fields_str):
-    str_data = custom_fields_str.strip().split(',')
+    str_data = split_fields(custom_fields_str.strip())
     custom_fields = {}
     for data in str_data:
         param_data = data.split(':')
         custom_fields[param_data[0]] = ':'.join(param_data[1:])
     return custom_fields
 
 
+def split_fields(s):
+    result = []
```

Comment:
can you please add a dev note of how `s` should look like when you get here? this code is hard to follow